packages/flutter/lib/src/widgets/app.dart

```diff
@@ -678,6 +678,9 @@ class _WidgetsAppState extends State<WidgetsApp> implements WidgetsBindingObserv
   Locale _locale;
 
   Locale _resolveLocale(Locale newLocale, Iterable<Locale> supportedLocales) {
+    if (newLocale == null) {
```

Comment:
Should a similar change be made in `widgets/binding.dart`?

Reply:
I dont think it directly accesses locale properties and mostly just uses == comparison over the whole locale, or calls the _resolveLocale() so might not be necessary.